SourceS/sdl2_to_1_2_backports.h

```diff
@@ -264,6 +277,334 @@ SDL_CreateRGBSurfaceWithFormatFrom(void *pixels, Uint32 flags, int width, int he
 	return SDL_CreateRGBSurfaceFrom(pixels, flags, width, height, depth, rmask, gmask, bmask, amask);
 }
 
+//= BlitScaled backport from SDL 2.0.9.
+
+#define SDL_BlitScaled SDL_UpperBlitScaled

```

Comment:
nice work, but did you consider resolution switching instead? It probably will perform better on most devices that supports it.

Reply:
I haven't thought of that. It'ss already playing at full speed on my fairly underpowered device, so I've left a note for now.

Reply:
Ok, it probably is a benefit to have both options available, some devices might not change resolution in a smooth way and others may need the reduced load.